readme.md

```diff
@@ -0,0 +1,96 @@
+In this project, I built:
+
+A clone of a New York times article which can be found in
+https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html;
+In this project I:
+
+Used CSS selectors correctly;
+Used HTML elements box model (margin, padding, width, height);
+Used industry-standard tools (flexbox) to place elements in the page;
+Applied front-end best practices;
+Used CSS flexbox;
+Applied HTML and CSS linter errors detectors.
+Built with
+HTML and CSS.
+
+Live Demo
+https://erezfree29.github.io/new-york-times/
```

Comment:
HI Uryelah 
My demo doesn't show all the sections as the branch is not merged to the main and I thought I need to present the main.
My question is if I change the code today will you still review it tomorrow?otherwise I will be forced to get a week's extension.